pkg/odo/cli/service/operator_backend.go

```diff
@@ -68,50 +74,26 @@ func (b *OperatorBackend) ValidateServiceCreate(o *CreateOptions) (err error) {
 			return err
 		}
 
-		err = yaml.Unmarshal(fileContents, &d.OriginalCRD)
-		if err != nil {
-			return err
-		}
-
-		// Check if the operator and the CR exist on cluster
-		var csv olm.ClusterServiceVersion
-		b.CustomResource, csv, err = svc.GetCSV(o.KClient, d.OriginalCRD)
+		err = yaml.Unmarshal(fileContents, &u.Object)
 		if err != nil {
 			return err
 		}
 
-		// all is well, let's populate the fields required for creating operator backed service
-		b.group, b.version, b.resource, err = svc.GetGVRFromOperator(csv, b.CustomResource)
-		if err != nil {
-			return err
-		}
+		gvk := svc.GetGVKFromUnstructured(u)
```

Comment:
Why not use function that is already in k8s apimachinery https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1/unstructured#Unstructured.GroupVersionKind ?

Reply:
Because I didn't check that it exists already. 😕

Reply:
https://github.com/openshift/odo/pull/4964/commits/8270bb639dbd38589082f73b590955c489cdddd7